changelog/_unreleased/2025-02-18-set-primary-image-of-page.md

```diff
@@ -0,0 +1,6 @@
+---
+title:              Set primary image of page for products
+issue:              NEXT-40675
+---
+# Storefront
+*  Set `primaryImageOfPage` property for products
```

Comment:
You have to use `Added`, `Changed` or  `Removed` for changelog entries.

But since we're trying to slowly get rid of these files, you could also remove this file and it would be okay for us